Makefile

```diff
@@ -8,7 +8,7 @@ GOMOD_RO_FLAG ?=
 
 build/linux/pentagon: $(DEPS)
 	mkdir -p build/linux
-	GOOS=linux go build $(GOMOD_RO_FLAG) -v -ldflags=$(LDFLAGS) -o ./build/linux/pentagon ./pentagon
+	GOOS=linux CGO_ENABLED=0 go build $(GOMOD_RO_FLAG) -v -ldflags=$(LDFLAGS) -o ./build/linux/pentagon ./pentagon
```

Comment:
While I generally support static-compilation, I feel like this should:
 1) be a different PR since it's logically distinct from the `SecretType` handling (and we'd roll them back independently if there was a problem with one of them)
 2) have a clear commit message explaining why we want to disable CGO, (which does end up with a statically compiled binary on Linux, but it isn't the only way (https://github.com/golang/go/issues/26492 should add a `-static` eventually, and has some snippets for fully static compilation))

Reply:
This change was removed and applied in another merge request.
#26492 is still not available. I ise the above-mentioned way which created an operatinal binary to be use din the selected alpine base container.